src/poker-evaluator.ts

```diff
@@ -8,6 +8,11 @@ import ThreeCardConverter from './three-card-converter';
 // This is outside the class so evalHand is static, to keep same api as @chenosaurus/poker-evaluator
 const RANKS_DATA = fs.readFileSync(path.join(__dirname, '../data/HandRanks.dat'));
 
+// Create a wrapper function to keep same api as @chenosaurus/poker-evaluator
+export function evalHand(cards: string[] | number[]) {
+  return PokerEvaluator.evalHand(cards);
+}
+
```

Comment:
Thanks for this. Actually what this has uncovered is there is no need for the class anymore, it adds an extra layer which needs deconstructing on the require (hidden by the es6 import)

Reply:
I guess the best course of action might be to reject the pr, and get rid of the class.